platforms/platform_db/discrete_pcie3_hssi40.json

```diff
@@ -50,6 +51,11 @@
             "interface": "avalon_mm_legacy_wires_2bank",
             "optional": true,
             "define": [ "INCLUDE_DDR4" ]
+         },
+
+         {
+            "class": "hssi",
+            "interface": "raw_pr"
```

Comment:
What is pr?

Reply:
Partial reconfiguration. The data structure is pr_hssi_if.

Ultimately, this is an interface class we don't expect many customers to use. We are expecting most will let us instantiate a MAC in the PIM and expose Avalon streams for the data. The "raw_pr" mode lets the current MACs work.